app/controllers/admin_controller.rb

```diff
@@ -60,22 +60,22 @@ def reset_user_password
     end
   end
 
-  def useremail 
-   if logged_in_as(['admin', 'moderator']) 
-     if params[:address] 
-       # address was submitted. find the username(s) and return. 
-       @address = params[:address] 
+  def useremail
+   if logged_in_as(['admin', 'moderator'])
+     if params[:address]
+       # address was submitted. find the username(s) and return.
+       @address = params[:address]
        if params[:include_banned]
-         @users = User.where(email: params[:address]) 
+         @users = User.where(email: params[:address])
            .where('created_at > (?)', DateTime.new(2015)) # since 2015, whether banned or not
        else
-         @users = User.where(email: params[:address]) 
-           .where(status: [1, 4]) 
+         @users = User.where(email: params[:address])
+           .where(status: [1, 4])
        end
-     end 
-   else 
-     # unauthorized. instead of return ugly 403, just send somewhere else 
-     redirect_to '/dashboard' 
+     end
+   else
+     # unauthorized. instead of return ugly 403, just send somewhere else
+     redirect_to '/dashboard'
```

Comment:
The tests are passing now @BondaSaim 🎉 

Are these changes needed? I am assuming that line 208 is the required change?

Reply:
Thank you @RuthNjeri.
Yes, The change at line 208 is the required one, the other are auto generated by my IDE

Reply:
@BondaSaim can you revert these changes to what was there before? Your current IDE seems to be using an indentation of 1 space instead of the recommended 2

Reply:
That's done